src/automoji.py

```diff
@@ -100,3 +104,23 @@ async def react_user_emoji(self, message: nextcord.Message):
                 await message.add_reaction(em)
             except nextcord.DiscordException as e:
                 self.logger.warning(e)
+
+    async def add_quote(self, message: nextcord.Message):
+        # Sends an error message if the channel isn't set.
+        self.cur.execute(
```

Comment:
Can you add this check in the on_message co-routine? If the quotechannel isn't None, and the current channel is the quotechannel, call the add_quote co-routine.